### PR TITLE
[dashboards] Implement support for Change widgets using formulas and functions

### PR DESCRIPTION
**Pull request by DrkSephy**
This pull request implements support for formulas and functions in the `Change` dashboard widget.

### CODE REVIEW

**Review by jtappa — DISMISSED** [review]
👍🏻  from docs team

**Review by jodiely — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-datadog
 
 require (
-	github.com/DataDog/datadog-api-client-go v1.6.0
+	github.com/DataDog/datadog-api-client-go v1.6.1-0.20211111153236-e7664265bffa
```
Comment (zippolyte) [inline go.mod:4]: Commit is too old, can you upgrade go client again ?

Comment (DrkSephy) [reply go.mod]: @zippolyte I've gone ahead and updated the go client.

**Review by jodiely — DISMISSED** [review]

**Review by zippolyte — DISMISSED** [review]

**Review by zippolyte — APPROVED** [review]

**Review by zippolyte — APPROVED** [review]

**Pull request merged**